client/my-sites/stats/stats-subscribers-chart-section/index.tsx

```diff
@@ -160,10 +160,10 @@ export default function SubscribersChartSection( {
 						<StatsPeriodHeader>
 							<Intervals selected={ period } pathTemplate={ pathTemplate } compact />
 						</StatsPeriodHeader>
-						<div className="subscribers-section-legend" ref={ legendRef }></div>
 					</div>
 				</div>
 			</div>
+			<div className="subscribers-section-legend" ref={ legendRef }></div>
```

Comment:
Could we move this line to be wrapped with `<UplotChart />` below, as their structure would be more meaningful to stay together? From doing that, this section would only appear when the chart is loaded and has data.